src/routes/AuthorityViewRoute/AuthorityViewRoute.js

```diff
@@ -64,7 +80,15 @@ const AuthorityViewRoute = () => {
   const authority = useAuthority({ recordId: id, authRefType, headingRef }, {
     onError: handleAuthorityLoadError,
   });
-  const marcSource = useMarcSource({ recordId: id, tenantId, enabled: Boolean(selectedAuthority) }, {
+
+  const isConsortiaEnv = stripes.hasInterface('consortium');
+  const isShared = selectedAuthority?.shared;
```

Comment:
Consider `!!(selectedAuthority?.shared)` or `Boolean(selectedAuthority?.shared)` to convert the `undefined` value returned by optional chaining to boolean false. Related, `selectedAuthority.shared` should really be `...isShared`, but I don't know if you're in charge of that property.